cas-server-support-ldap/src/main/java/org/jasig/cas/authentication/LdapAuthenticationHandler.java

```diff
@@ -18,6 +18,8 @@
  */
 package org.jasig.cas.authentication;
 
+import com.google.common.base.Functions;
+import com.google.common.collect.Maps;
```

Comment:
When did Guava components become available? We can do all sorts of fun collection magic now :)